src/pyfmi/fmi.pyx

```diff
@@ -575,6 +578,7 @@ cdef class ModelBase:
                 file_path -- path to extracted XML file
             otherwise function returns nothing
         """
+        # XXX: Needs to be here for now, due to cicular dependency, since pyfmi.common.log.parser imports FMUException (from this file)
```

Comment:
```suggestion
        # XXX: Needs to be here for now, due to circular dependency, since pyfmi.common.log.parser imports FMUException (from this file)
```